src/map/pc.hpp

```diff
@@ -787,6 +787,7 @@ class map_session_data {
 		uint32 pending_weight;
 		uint32 pending_zeny;
 		uint16 pending_slots;
+		uint32 target_id;
```

Comment:
As it is called dest_id in the other structure, it would be nice to follow that naming.